source/codegen/metadata/nixnetsocket/functions.py

```diff
@@ -1,4 +1,29 @@
 functions = {
+    'Accept': {
```

Comment:
Besides Accept there were `RecVFrom`, `GetSockName`, and 'GetPeerName' that have these outputs. I'm happy to add those after this PR goes in I have the metadata already written.

Reply:
Yeah, I just noticed that and split the task. So now the one for remaining socket outputs is in the New state. We may also need to double-check that there isn't any funny business with those methods.